docs/components-overview.md

```diff
@@ -48,42 +48,44 @@ The following table lists the .NET Aspire components currently available for use
     # [PackageReference](#tab/package-reference)
 
     ```xml
-    <PackageReference Include="Aspire.Npgsql"
-                      Version="[SelectVersion]" />
+    <PackageReference Include="Aspire.Npgsql" Version="[SelectVersion]" />
     ```
 
     ---
 
     For more information, see [dotnet add package](/dotnet/core/tools/dotnet-add-package) or [Manage package dependencies in .NET applications](/dotnet/core/tools/dependencies).
 
-1. In the _Program.cs_ file of your project, call the <xref:Microsoft.Extensions.Hosting.AspirePostgreSqlNpgsqlExtensions.AddNpgsqlDataSource%2A> extension method to register a `NpgsqlDataSource` for use via the dependency injection container. The method expects a connection name parameter.
+1. In the _Program.cs_ file of your Worker Service project, call the <xref:Microsoft.Extensions.Hosting.AspirePostgreSqlNpgsqlExtensions.AddNpgsqlDataSource%2A> extension method to register `NpgsqlDataSource` as a service.
 
-    ```csharp
-    builder.AddNpgsqlDataSource("PostgreSqlConnection");
-    ```
+    :::code source="snippets/components/AspireApp/WorkerService/Program.cs" highlight="5":::
 
-    > [!NOTE]
-    > Components that are designed to connect to Azure services also support passwordless authentication and authorization using Azure RBAC, which is the recommended approach for production apps.
+    The preceding code adds the `NpgsqlDataSource` to the dependency injection container with the connection name of `"customers"`. The connection name is later used by the orchestrator project, when expressing resource dependencies.
```

Comment:
Should there be quotes in a code block? It kind of gets the point across without them. Consider describing a normal string. You wouldn't say:

> Set your string to `"Going to the store"`

You would just do

> Set your string to `Going to the store`

or

> Set your string to "Going to the store"

Or maybe you would and I'm thinking about this wrong and it's ME who should change?!?! Tell me please 😱

Reply:
I'm not really consistent with this and I really should be. I strive to refer to things as they are, for example, when I'm referring to a code element that is a string with an exact value, I tend to call it that. It's the argument that satisfies the parameter, and its value is `"customers"`. What do you typically do?